### DISCUSSION TITLE
DAX Studio show name of connection

### DISCUSSION THREAD

**Discussion by emirtut** [DaxStudio/DaxStudio#1161] [Q&A] [answered]
Is there feature to show in DAX studio name of file which is connected to DAX Studio?
This will be valuable info for history of querying, of course as option.

**Comment by dgosbell** [comment, marked as answer]
No, DAX Studio does not connect to files. Even when you connect to Power BI Desktop we connect to the running data model and pull the model name from the title bar of the Power BI Desktop instance, we don't know where the actual file is located.
